app/build.gradle

```diff
@@ -66,5 +67,11 @@ dependencies {
     testImplementation "junit:junit:4.12"
     androidTestImplementation "androidx.test:runner:1.2.0"
     androidTestImplementation "androidx.test.espresso:espresso-core:3.2.0"
+
+    // Room
+    implementation "android.arch.persistence.room:runtime:1.1.1"
+    kapt "android.arch.persistence.room:compiler:1.1.1"
+
+
 }
 apply plugin: 'com.google.gms.google-services'

```

Comment:
Can you do me a favour and move these to the top of the file with the rest of them